<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>403</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>369</width>
    <height>403</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Minifier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/pics/img/icon.png</normaloff>:/pics/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="minimumSize">
    <size>
     <width>348</width>
     <height>353</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: #fff;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>341</width>
        <height>351</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget{
	background-color: #fff;
}

QWidget{
	background-color: #fff;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="codeTab">
       <property name="styleSheet">
        <string notr="true">QWidget{
color:#01293d;
}</string>
       </property>
       <attribute name="title">
        <string>Compresseur de code</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="titleCode">
          <property name="font">
           <font>
            <family>Meiryo UI</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:#01293d;
}</string>
          </property>
          <property name="text">
           <string>Compresseur de code:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="jsCheck">
          <property name="font">
           <font>
            <family>Meiryo UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox{
color:#01293d;
}
QCheckBox::indicator:checked
{
height: 25px;
width: 25px;
background-color: #0b94d1;
background-image: url(:/pics/img/check.png);
background-repeat:no-repeat;
background-position:center center;
}
QCheckBox::indicator:unchecked
{
height: 25px;
width: 25px;
background-color: #eee;
}
QCheckBox::indicator:checked:hover
{
background-color: rgb(9, 136, 190);
}
QCheckBox::indicator:unchecked:hover
{
background-color: rgb(203, 203, 203);
}
QCheckBox::indicator:checked:pressed
{
background-color: rgb(9, 136, 190);
}
QCheckBox::indicator:unchecked:pressed
{
background-color: rgb(203, 203, 203);
}</string>
          </property>
          <property name="text">
           <string>   JS</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="js_browser">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#fff;
border:2px solid #0b94d1;</string>
          </property>
          <property name="text">
           <string>Choisir le chemin JS</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="jsPath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="cssCheck">
          <property name="font">
           <font>
            <family>Meiryo UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox{
color:#01293d;
}
QCheckBox::indicator:checked
{
height: 25px;
width: 25px;
background-color: #0b94d1;
background-image: url(:/pics/img/check.png);
background-repeat:no-repeat;
background-position:center center;
}
QCheckBox::indicator:unchecked
{
height: 25px;
width: 25px;
background-color: #eee;
}
QCheckBox::indicator:checked:hover
{
background-color: rgb(9, 136, 190);
}
QCheckBox::indicator:unchecked:hover
{
background-color: rgb(203, 203, 203);
}
QCheckBox::indicator:checked:pressed
{
background-color: rgb(9, 136, 190);
}
QCheckBox::indicator:unchecked:pressed
{
background-color: rgb(203, 203, 203);
}</string>
          </property>
          <property name="text">
           <string>   CSS</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="cssPath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="css_browser">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#fff;
border: 2px solid #0b94d1;</string>
          </property>
          <property name="text">
           <string>Choisir le chemin CSS</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="launchCode">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 2px solid #0b94d1;
border-radius:20px;
color:#01293d;
}
QPushButton:hover{
background-color: #0b94d1;
color: white;
}</string>
          </property>
          <property name="text">
           <string>Compresser</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="imgTab">
       <property name="styleSheet">
        <string notr="true">QWidget{
color:#01293d;
}</string>
       </property>
       <attribute name="title">
        <string>Compresseur d'images</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
          <item>
           <widget class="QLabel" name="labelPath">
            <property name="font">
             <font>
              <family>Meiryo UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
color:#01293d;
}</string>
            </property>
            <property name="text">
             <string>Image :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <widget class="QTextEdit" name="fileName">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextEdit{
background-color:#fff;
border:0 solid #0b94d1;
border-bottom: 2px solid #0b94d1;
}</string>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="placeholderText">
             <string>nom fichier transformer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="titleImg">
          <property name="font">
           <font>
            <family>Meiryo UI</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:#01293d;
}</string>
          </property>
          <property name="text">
           <string>Compresseur d'images:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QSlider" name="qualitySlider">
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
border: 1px solid #01293d;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #0b94d1, stop: 1 rgb(12, 177, 247));
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 rgb(12, 177, 247), stop: 1 #0b94d1);
border: 1px solid #01293d;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #01293d;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #01293d;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #01293d;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #01293d;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #01293d;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #0b94d1;
border-radius: 4px;
}</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QPushButton" name="loadImg">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 2px solid #0b94d1;
background-color: #0b94d1;
border-radius:20px;
color: white;
}
QPushButton:hover{
background-color: #fff;
color:#01293d;
}</string>
          </property>
          <property name="text">
           <string>Charger une image</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="qualtiySpinBox">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QPushButton" name="launchImage">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 2px solid #0b94d1;
border-radius:20px;
color:#01293d;
}
QPushButton:hover{
background-color: #0b94d1;
color: white;
}</string>
          </property>
          <property name="text">
           <string>Compresser</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelQuality">
          <property name="font">
           <font>
            <family>Meiryo UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:#01293d;
}</string>
          </property>
          <property name="text">
           <string>Qualité :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="create_config"/>
    <addaction name="load_config"/>
    <addaction name="manage_config"/>
   </widget>
   <widget class="QMenu" name="menuCompresseur">
    <property name="title">
     <string>Compresseur</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuConfig"/>
   <addaction name="menuCompresseur"/>
  </widget>
  <action name="create_config">
   <property name="text">
    <string>Ajouter une sauvegarde</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="load_config">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Charger une sauvegarde</string>
   </property>
  </action>
  <action name="manage_config">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gérer les sauvegarde</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
